src/main/kotlin/org/jetbrains/intellij/platform/gradle/tasks/ComposedJarTask.kt

```diff
@@ -64,7 +71,8 @@ abstract class ComposedJarTask : Jar() {
                 duplicatesStrategy = DuplicatesStrategy.EXCLUDE
                 JarCompanion.applyPluginManifest(this)
 
-                project.artifacts.add(intellijPlatformComposedJarConfiguration.name, this)
+                intellijPlatformComposedJarConfiguration.outgoing.artifact(this)
+                intellijPlatformComposedJarApiConfiguration.outgoing.artifact(this)
```

Comment:
I noticed that https://github.com/JetBrains/intellij-platform-gradle-plugin/blob/4633113b553733c46656efb6f69ece7ed8093441/src/main/kotlin/org/jetbrains/intellij/platform/gradle/tasks/JarSearchableOptionsTask.kt#L82 creates jar for searchable options in runtime elements. I am not sure if that is how it should be, since we have these custom configurations and all of a sudden that thing is in the default one.

Reply:
May be related #1802

Reply:
probably related, yes.